calico_kubernetes/calico_kubernetes.py

```diff
@@ -141,6 +137,9 @@ def status(self, namespace, pod_name, docker_id):
                 workload_id=self.docker_id
             )
         except KeyError:
+            # Obtain information from Docker Client and validate container state
+            # Will exit safely (0) in host NetworkMode
+            self._validate_container_state(self.docker_id)
```

Comment:
Sounds good.  How does the kubelet behave when it receives a exit code of 0, but no status is returned via stdout?  Is that the expected behavior?